pkg/adapter/adapter.go

```diff
@@ -162,6 +163,9 @@ func (a *Adapter) PollForMessages(stopCh <-chan struct{}) error {
 			queue, err = a.channel.QueueInspect(a.config.QueueName)
 			if err != nil {
 				logger.Error(err.Error())
+				a.rmqHelper.CloseRabbitMQConnections(a.connection, logger)
+				go a.rmqHelper.WaitForRetrySignal()
```

Comment:
hacky part for it to wait one second, flush the retrychannel that start to get retry signals each time we retry